lib/helpers/getUserReport.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var timeHelper = require('./time');
+
+module.exports = function getUserReport(standup) {
+  var color = '#000000'.replace(/0/g, function () {
+    return (~~(Math.random()*16)).toString(16);
+  });
+  var fields = [];
+  if (standup.yesterday) {
+    fields.push({
+        title: 'Yesterday',
+        value: standup.yesterday,
+        short: false
+    });
+  }
+  if (standup.today) {
+    fields.push({
+      title: 'Today',
+      value: standup.today,
+      short: false
+    });
+  }
+  if (standup.blockers) {
+    fields.push({
+      title: 'Blockers',
+      value: standup.blockers,
+      short: false
+    });
+  }
+  if (standup.goal) {
+    fields.push({
+      title: 'Goal',
+      value: standup.goal,
+      short: false
+    });
+  }
+
+  return {
+    title: timeHelper.getReportFormat(standup.date),
+    fields: fields,
+    color: color,
+    thumb_url: standup.thumbUrl
+  };
+};
```

Comment:
Also not sure. I'm going to leave this for now, but will add an issue to look at it.